.github/workflows/daily-build.yml

```diff
@@ -0,0 +1,38 @@
+name: Daily Build
+
+on:
+  schedule:
+    - cron:  '0 */12 * * *'
```

Comment:
Let's increase the frequency of this job

Reply:
Fixed in 823fd0e51f8f8a460d5e83b3c1026d65520a7a16

Reply:
Updated to every 8 hours in 81c27da9fbdc7f8ceaaa80228a825f2afd8a13e7